Objective-C/CBLBlob.mm

```diff
@@ -184,6 +189,11 @@ - (NSData*) content {
         FLSliceResult_Release(res);
         if (content && content.length <= kMaxCachedContentLength)
             _content = content;
+        
+        // update the content and length while loading it
+        [self updateProperties: kLengthMetaProperty value: @(_content.length)];
+        [self updateProperties: kDataMetaProperty value: _content];
```

Comment:
We don't need to put _content into the dictionary. As we don't need to keep _properties, we will just update _length directly (make sure to synchronize).